app/assets/javascripts/social_share_button.js

```diff
@@ -0,0 +1,59 @@
+// Opening pop-ups with share url
+function openShareUrl(url, initialWidth = 640, initialHeight = 480) {
+  if (typeof url !== "string" || !url.startsWith("http")) {
+    console.error("Invalid URL");
+    return;
+  }
+
+  const width = Math.max(100, Math.min(screen.width, initialWidth));
+  const height = Math.max(100, Math.min(screen.height, initialHeight));
+
+  const left = (screen.width / 2) - (width / 2);
+  const top = (screen.height * 0.3) - (height / 2);
+  const opts = `width=${width},height=${height},left=${left},top=${top},menubar=no,status=no,location=no`;
+
+  const newWindow = window.open(url, "popup", opts);
+
+  if (!newWindow || newWindow.closed || typeof newWindow.closed == "undefined") {
+    console.error("Popup blocked. Please allow popups for this website.");
```

Comment:
Who is supposed to be able to read this message? Regular users don't look at their browser console.

Reply:
Removed this.